schema_test.go

```diff
@@ -47,7 +47,7 @@ func TestSchema_CreateTable(t *testing.T) {
 		Name:     "wishlists",
 		Optional: true,
 		Definitions: []TableDefinition{
-			Column{Name: "id", Type: ID},
+			Column{Name: "id", Type: ID, Primary: true},
```

Comment:
looks like this is a breaking changes? if primary is not specified, then all id won't be treated as primary?
and what happens if type other than ID specified as Primary?

Reply:
if you use `t *rel.Table` like:

```go
t.BigID("id")
```
than it is not breaking change as `(*Table) BigID` and `(*Table) ID` will automatically set `Primary` as true but if you use:

```go
t.Column("id", rel.BigID)
```

than it will be breaking change and you will have to manually add `Primary` now

Reply:
See here: https://github.com/go-rel/rel/pull/247/files#diff-134c48f173480a9772719053c1b3e967756a71c75f84ed4f94f12c71cd91ed30R26

Reply:
I moved change to `(*Table) Column` so it should be not breaking in any case now, except if you manually add `Definitions` to `Table`

Reply:
got it 👍

Reply:
Sorry for resolving too quickly, seems this question is still not answeres?
> and what happens if type other than ID specified as Primary?

Reply:
Just as currently you can specify any column as primary, you have to manually assign unique value or values to it

Reply:
Or I misunderstood your question...

Reply:
I see, that means we will also support primary declaration such as this: `t.String("uuid", rel.PrimaryKey(true))` right?
generated sql will be: `uuid varchar(225) primary key`

Reply:
yes